### PR TITLE
Renata/improve osm imports

### PR DESCRIPTION
**Pull request by r-akemii**

### CODE REVIEW

**Comment by pedrocamargo** [comment]
There is an empty project_database.sqlite in the root of the package that probably shouldn;t be there

image

File: tests/create_nauru_test.py [hunk 1]
```diff
@@ -1,12 +1,13 @@
-from aequilibrae.utils.create_example import create_example
-from shapely.geometry import Point
-import shapely.wkb
 from math import sqrt
-import geopandas as gpd
 from os.path import dirname, join
-from tradesman.model_creation.create_new_tables import add_new_tables
-from tradesman.model_creation.import_population import import_population
+
```
Comment (pedrocamargo) [inline tests/create_nauru_test.py:3]: Optimized imports. Very nice!

File: tests/create_nauru_test.py [hunk 2]
```diff
@@ -60,22 +59,9 @@ def create_nauru_test(folder):
...
-
-    pop_per_zone = pop_to_zone.groupby(["zone_id"]).sum()[["population"]].reset_index()
-    pop_per_zone.loc[:, "zone_id"] = pop_per_zone.zone_id.astype(int)
-    pop_per_zone.sort_values(["zone_id"], inplace=True)
-
-    zones_with_pop = zones_from_location.merge(pop_per_zone, on="zone_id", how="left")
-    zones_with_pop.population.fillna(0, inplace=True)
-
-    zones_with_pop = zones_with_pop.drop_duplicates(subset=["geom"])
+    population = [358, 686, 541, 966, 474, 158, 1088, 214, 1178, 473, 59, 1004, 186, 499, 331, 199, 763, 447, 222]
```
Comment (pedrocamargo) [inline tests/create_nauru_test.py:62]: Good simplification

File: tests/model_creation/test_user_import_new_totals.py [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
 from tempfile import gettempdir, mkdtemp
-import unittest
 
 from tradesman.model_creation.synthetic_population.user_control_import import user_import_new_totals
 
 
 class TestUserImportNewTotals(unittest.TestCase):
     def setUp(self) -> None:
-        temp_src = mkdtemp()
+        temp_src = mkdtemp(dir=gettempdir())
```
Comment (pedrocamargo) [inline tests/model_creation/test_user_import_new_totals.py:12]: This pattern seems a bit too convoluted. Doesn't it?

**Comment by r-akemii** [comment]
@pedrocamargo I fixed the errors related to the merge conflict. The test to run the entire model (_test_model.py_) is skipped because it returned a memory usage related error (exit code 137), possibly due to the size of some files being downloaded. Thus, the existing tests are only testing the workflow, but not whether the links/APIs are working correctly.

**Review by pedrocamargo — APPROVED** [review]

**Pull request merged**